RunII_106X_v2/SM/UL17/TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1.xml

```diff
@@ -22,13 +22,15 @@
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_32.root" Lumi="0.0"/>
-<In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_4.root" Lumi="0.0"/>
+<!--EMPTY <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_4.root" Lumi="0.0"/> -->
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_5.root" Lumi="0.0"/>
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_6.root" Lumi="0.0"/>
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_7.root" Lumi="0.0"/>
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_8.root" Lumi="0.0"/>
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220207_115904/0000/Ntuple_9.root" Lumi="0.0"/>
 <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220211_100050/0000/Ntuple_1.root" Lumi="0.0"/>
-<In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220211_100050/0000/Ntuple_2.root" Lumi="0.0"/>
+<!--EMPTY <In FileName="/pnfs/desy.de/cms/tier2//store/group/uhh/uhh2ntuples/RunII_106X_v2/UL17/TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/crab_TTWJetsToQQ_CP5_amcatnloFXFX-madspin-pythia8_Summer20UL17_v1/220211_100050/0000/Ntuple_2.root" Lumi="0.0"/> -->
 <!-- < NumberEntries="655018" Method=fast /> -->
 <!-- < NumberEntries="444333.97475" Method=weights /> -->
+<!-- < NumberEntries="584236" Method=fast /> -->
```

Comment:
There is something wrong here, isn't it? The old counting results were not removed.

Reply:
Right, these are still the pre loss counts. I must have missed that one. Good catch!

Reply:
is this fine now?

Reply:
Looks fine to me